admin/services/score.service.js

```diff
@@ -0,0 +1,44 @@
+const checkDataService = require('./data-access/check.data.service')
+
+const scoreService = {}
+
```

Comment:
is this a completely new file?  as i added something similar a while back...

Reply:
let me know which one it is please.